internal/testing/testdata/testdata.go

```diff
@@ -578,6 +578,15 @@ var (
 				Justification:  isDepJustifyContainedByJustification,
 			},
 		},
+		{
+			Pkg:    keysPack,
+			DepPkg: rsaPubFilePack,
+			IsDependency: &model.IsDependencyInputSpec{
+				DependencyType: model.DependencyTypeUnknown,
+				VersionRange:   "",
+				Justification:  isDepJustifyContainsJustification,
+			},
+		},
```

Comment:
@lumjjb seen anything like this with the new SPDX library update?

Reply:
there is a bit of a bug in 0.5.0 which has been fixed in 0.5.1 assuming that this was caused by the shortcut fields (https://github.com/spdx/tools-golang/issues/208)

Reply:
ah ok it looks like that bug wasn't fully fixed because we didnt check for the CONTAINED_BY relationship as a point of duplicates. I think we can keep this for now and this will be fixed later on

Reply:
fixed, and released in https://github.com/spdx/tools-golang/releases/tag/v0.5.2